src/cursor/cursor.ts

```diff
@@ -24,6 +24,8 @@ import type { Sort, SortDirection } from '../operations/find';
 import type { Hint, OperationBase } from '../operations/operation';
 import type { Document } from '../bson';
 
+const kCursor = Symbol('internalCursor');
```

Comment:
nit: I would just call this `cursor`, doesn't hugely matter but since the purpose of this is to make it easy to print out/inspect maybe brevity is preferred